docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -0,0 +1,155 @@
+- **Percent -** Available when **Stack** is selected. Each series is drawn as a percentage of the total of all series.
+- **Null value -** How null values are displayed.
+  - **connected -** XXX
+  - **null -** XXX
+  - **null as zero -** XXX
+
+## Series overrides
+
+Series overrides allows a series in a graph panel to be rendered differently from the others. You can customize display options on a per-series bases or by using regex rules. For example, one series can have a thicker line width to make it stand out or be moved to the right Y-axis.
+
+You can add multiple series overrides.
+
+**Add a series override**
+
+1. Click **Add series override**.
+1. In **Alias or regex** Type or select a series. Click in the field to see a list of available series.
+1. Click **+** and then select a style to apply to the series. You can add multiple styles to each entry.
+  - **Bars -** Show series as a bar graph.
+  - **Lines -** Show series as line graph.
+  - **Line fill -** Show line graph with area fill.
+  - **Fill gradient -** Area fill gradient amount.
+  - **Line width -** Set line width.
+  - **Null point mode -** Option to ignore null values or replace with zero. Important if you want to ignore gaps in your data.
+  - **Fill below to -** Fill area between two series.
+  - **Staircase line -** Show series as a staircase line .
+  - **Dashes -** Show line with dashes.
+  - **Hidden Series -** Hide the series.
+  - **Dash Length -** Dashed line length.
+  - **Dash Space -** Dashed line spacing.
+  - **Points -** Show series as separate points.
+  - **Point Radius -** Radius for point rendering.
+  - **Stack -** Set stack group for series.
+  - **Color -** Set series color.
+  - **Y-axis -** Set series y-axis.
+  - **Z-index -** Set series z-index (rendering order). Important when overlaying different styles (bar charts, area charts).
+  - **Transform -** Transform value to negative to render below the y-axis.
+  - **Legend -** Control if a series is shown in legend.
+  - **Hide in tooltip -** Control if a series is shown in graph tooltip.
+
+## Axes
+
+Use these options to control the display of axes in the visualization.
+
+### Left Y/Right Y
+
+Options are identical for both Y-axes.
+
+- **Show -** Click to show or hide the axis.
+- **Unit -** The display unit for the Y value.
+- **Scale -** The scale to use for the Y value, linear, or logarithmic. (default linear)
+- **Y-Min -** The minimum Y value. (default auto)
+- **Y-Max -** The maximum Y value. (default auto)
+- **Decimals -** Defines how many decimals are displayed for Y value. (default auto)
+- **Label -** The Y axis label. (default “")
+
+### Y-Axes
+- **Align -** Select to align left and right Y-axes by value. (default unchecked/false)
+- **Level -** Available when **Align** is selected. Value to use for alignment of left and right Y-axes, starting from Y=0. (default 0)
+
+### X-Axis
+
+- **Show -** Click to show or hide the axis.
+- **Mode -** XXX
```

Comment:
Mode: The display mode completely changes the visualization of the graph panel. It's like three panels in one. The main mode is the time series mode with time on the x-axis. The other two modes are a basic bar chart mode with series on the x-axis instead of time and a histogram mode.

- Value: The aggregation type to use for the values. The default is total (summing the values together).

- Buckets: The number of buckets to group the values by. If left empty, then Grafana will try and calculate a suitable number of buckets.
- X-min: filters out values from the histogram that are under this minimum limit.
- Y-min: filters out values that are greater than this maximum limit.